### PR TITLE
add project URLs to setup.py

### PR DESCRIPTION
**Pull request by sappelhoff**
fix for https://github.com/mne-tools/mne-python/blob/071a98fae28bd7c32c6ce357076263ef01031994/doc/sphinxext/related_software.py#L78

- xref https://github.com/mne-tools/mne-python/pull/12731

cc @larsoner @damian5710

### CODE REVIEW

**Comment by sappelhoff** [comment]
or perhaps @ephathaway or @mscheltienne who seem to have committed to this repo before :)

File: setup.py [hunk 1]
```diff
@@ -31,4 +31,9 @@
         "Operating System :: OS Independent",
     ],
     python_requires='>=3.6',
+    project_urls={
+        'Homepage': 'https://github.com/BEL-Public/mffpy',
+        'Bug Reports': 'https://github.com/BEL-Public/mffpy/issues',
+        'Source': 'https://github.com/BEL-Public/mffpy',
+    },
```
Comment (damian5710) [inline setup.py]:
```suggestion
    },
```

**Review by damian5710 — CHANGES_REQUESTED** [review]
Hi @sappelhoff ! Thanks for contributing to this project. It seems that there are trailing whitespaces in line 38. Could you remove them before merging this PR please?

**Comment by sappelhoff** [comment]
> it seems that there are trailing whitespaces in line 38. Could you remove them before merging this PR please?

of course! Thanks for spotting it and thanks for the code suggestion @damian5710.

**Review by damian5710 — APPROVED** [review]
Looks good! Thank you, @sappelhoff

**Comment by sappelhoff** [comment]
@damian5710 what points are left to do to get this merged?

**Comment by damian5710** [comment]
Sorry for the delay. I'll go ahead and merge this PR now. Thanks!

**Pull request merged**